server/api/controllers/signUpController.js

```diff
@@ -2,9 +2,8 @@
 // eslint-disable-next-line import/no-named-as-default
 import User from '../models/user';
 
-
 const SignUp = {
-	signup(req, res) {
+	assignup(req, res) {
```

Comment:
Can you change your editor tabulation's space to 2 spaces?

Reply:
Okay on it